plugins/web/include/web/configuration.hpp

```diff
@@ -21,7 +21,7 @@ struct configuration {
     static auto result = vast::record_type{
       {"bind", vast::string_type{}},    {"port", vast::int64_type{}},
       {"mode", vast::string_type{}},    {"certfile", vast::string_type{}},
-      {"keyfile", vast::string_type{}}, {"web-root", vast::string_type{}},
```

Comment:
Please don't remove the option to have a webroot at all, that's independent of the UI itself.

I think its realistic to expect this to still be useful for local development, to run against an artifact created from the closed-source CI, or longer-term for a developer-centric UI bundled with VAST itself.

Reply:
Fair point. I added this back.